package.json

```diff
@@ -55,72 +55,67 @@
   },
   "author": "",
   "dependencies": {
-    "@babel/runtime": "^7.12.5",
-    "@babel/runtime-corejs3": "^7.12.5",
-    "backoff-web": "^1.0.1",
-    "browserama": "^3.1.1",
-    "core-js": "^3.7.0",
+    "@babel/runtime": "^7.24.6",
+    "@babel/runtime-corejs3": "^7.24.6",
+    "browserama": "^3.2.2",
+    "core-js": "^3.37.1",
     "genesys-cloud-client-logger": "^4.2.12",
     "genesys-cloud-streaming-client": "^17.1.3",
-    "jwt-decode": "^3.1.2",
-    "lodash": "^4.17.15",
+    "jwt-decode": "^4.0.0",
+    "lodash": "^4.17.21",
     "process-fast": "^1.0.0",
-    "rxjs": "^7.5.5",
+    "rxjs": "^7.8.1",
     "safe-json-stringify": "^1.2.0",
-    "softphone-vendor-headsets": "^2.5.0",
+    "softphone-vendor-headsets": "^2.5.1",
     "strict-event-emitter-types": "^2.0.0",
     "uuid": "^9.0.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.3",
+    "@babel/core": "^7.24.6",
     "@babel/plugin-proposal-class-properties": "^7.12.1",
-    "@babel/plugin-transform-property-mutators": "^7.12.1",
-    "@babel/plugin-transform-runtime": "^7.12.1",
-    "@babel/preset-env": "^7.12.1",
-    "@babel/preset-typescript": "^7.12.1",
-    "@rollup/plugin-commonjs": "^22.0.0-1",
-    "@rollup/plugin-node-resolve": "^13.0.6",
-    "@types/jest": "^27.5.2",
-    "@types/lodash": "^4.14.165",
+    "@babel/plugin-transform-property-mutators": "^7.24.6",
+    "@babel/plugin-transform-runtime": "^7.24.6",
+    "@babel/preset-env": "^7.24.6",
+    "@babel/preset-typescript": "^7.24.6",
+    "@rollup/plugin-commonjs": "^25.0.8",
+    "@rollup/plugin-node-resolve": "^15.2.3",
+    "@types/jest": "^29.5.12",
+    "@types/lodash": "^4.17.4",
     "@types/nock": "^11.1.0",
-    "@types/node": "^15.12.0",
-    "@types/safe-json-stringify": "^1.1.0",
+    "@types/node": "^20.12.12",
     "@types/uuid": "^9.0.8",
-    "@types/webrtc": "^0.0.30",
-    "@types/ws": "^7.2.9",
-    "@typescript-eslint/eslint-plugin": "^4.29.3",
-    "@typescript-eslint/parser": "^4.29.3",
+    "@types/webrtc": "^0.0.43",
+    "@types/ws": "^8.5.10",
+    "@typescript-eslint/eslint-plugin": "^7.11.0",
+    "@typescript-eslint/parser": "^7.11.0",
     "axios-mock-adapter": "^1.22.0",
-    "babel-jest": "^27.5.1",
-    "babel-loader": "^8.0.6",
-    "babel-plugin-istanbul": "^6.0.0",
+    "babel-jest": "^29.7.0",
+    "babel-loader": "^9.1.3",
+    "babel-plugin-istanbul": "^6.1.1",
     "babel-preset-env": "^1.7.0",
     "babelify": "^10.0.0",
-    "broadcast-channel": "^4.9.0",
-    "codecov": "^3.8.1",
-    "concurrently": "^5.1.0",
+    "broadcast-channel": "^7.0.0",
+    "codecov": "^3.8.2",
+    "concurrently": "^8.2.2",
     "crypto": "^1.0.1",
-    "eslint": "^7.32.0",
-    "eslint-plugin-import": "^2.24.2",
-    "eslint-plugin-jsdoc": "^36.0.8",
-    "jest": "^27.5.1",
-    "npm-watch": "^0.11.0",
-    "pre-push": "^0.1.1",
+    "eslint": "^8.56.0",
```

Comment:
I didn't upgrade to version `9.3.0` because some of our typescript parsers aren't updated for that version yet. Version 9 will require some changes on our end too, we'll need to convert our `eslintrc.json` to `eslint.config.mjs` and use flat config as the default.